src/core/api/index.js

```diff
@@ -73,6 +73,7 @@ type CallApiParams = {|
   params?: Object,
   schema?: Object,
   state?: ApiStateType,
+  _config?: config,
```

Comment:
It doesn't matter too much since the result will be `Object` but this should be `_config?: typeof config`. You didn't get an error because the test files do not opt into Flow.